sphinx/builders/epub3.py

```diff
@@ -231,9 +231,9 @@ def setup(app):
     app.add_config_value('epub_theme', 'epub', 'html')
     app.add_config_value('epub_theme_options', {}, 'html')
     app.add_config_value('epub_title', lambda self: self.html_title, 'html')
-    app.add_config_value('epub_author', 'unknown', 'html')
+    app.add_config_value('epub_author', lambda self: self.author, 'html')
```

Comment:
This is same as what quickstart does.
https://github.com/sphinx-doc/sphinx/blob/77ae4249c62bebb137b29b1892a6b89b70c984b5/sphinx/templates/quickstart/conf.py_t#L173-L177

So I will be able to omit them from `conf.py`.